src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -374,6 +374,27 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
             }
         }
 
+        [Fact]
+        public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
+        {
+            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+            const string SlowStreamingServer = "http://httpbin.org/drip?numbytes=8192&duration=15&delay=1&code=200";
```

Comment:
FWIW, it currently takes ~20 seconds for me, when passing, on both Windows and Linux.